pkg/hostap/monitor.go

```diff
@@ -313,6 +352,13 @@ func (m *InterfaceMonitor) writeCommand(command string) error {
 func (m *InterfaceMonitor) handleAuthenticateEvent(addr string) error {
 	m.addrMutex.Lock()
 	defer m.addrMutex.Unlock()
+	if len(m.authenticatedAddrs) == 0 {
+		m.PfInfo.Authenticated = true
+		err := m.PfInfo.ConfigureVlanStateForVFs()
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
I would vote for moving this whole block into          `trafficcontrol.AllowTrafficFromMac(m.IfName, addr)`

Reply:
yes, it makes sense. moved this logic into `trafficcontrol.AllowTrafficFromMac `method.